Wireframe/index.html

```diff
@@ -1,13 +1,67 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Wireframe</title>
```

Comment:
Would have been nice to change the title, as this is what appears on the tab at the top of the browser. Also good for SEO (Search Engine Optimisation) !